geomesa-accumulo/geomesa-accumulo-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -59,6 +57,14 @@ declare -a urls=(
   "${base_url}org/apache/htrace/htrace-core4/${htrace4_core_version}/htrace-core4-${htrace4_core_version}.jar"
 )
 
+accumulo_maj_ver="$(expr match "$accumulo_version" '\([0-9][0-9]*\)\.')"
+if [[ "$accumulo_maj_ver" -lt 2 ]]; then
+  urls+=(
+    "${base_url}org/apache/accumulo/accumulo-fate/${accumulo_version}/accumulo-fate-${accumulo_version}.jar"
```

Comment:
Accumulo-{fate,trace} are gone in 2.0?

Reply:
yeah. although i notice that there is an `accumulo-tracer-2.0.0.jar` now... possibly only needed if you run remote traces? i'm not entirely sure what the old trace jar was used for either.

Reply:
@jahhulbert @jahhulbert-ccri any thoughts or info on the use of Accumulo Tracer?